### PR TITLE
Switch to glog for Volume API logging

### PR DESCRIPTION
**Pull request by prateekpandey14**
**What this PR does / why we need it**:
This commit will add glog as a default logger pkg for
volume API's for proper logging.

**Which issue this PR fixes** :
 fixes # openebs/openebs#265
ref - openebs/mayaserver#77

### CODE REVIEW

File: cmd/maya-apiserver/app/server/volume_endpoint.go [hunk 1]
```diff
@@ -5,31 +5,32 @@ import (
...
+		return s.VolumeSpecificGetRequest(resp, req)
 	default:
 		return nil, CodedError(405, ErrInvalidMethod)
 	}
 }
 
-// vsmSpecificGetRequest deals with HTTP GET request w.r.t a single VSM
-func (s *HTTPServer) vsmSpecificGetRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+// VolumeSpecificGetRequest deals with HTTP GET request w.r.t a single Volume
+func (s *HTTPServer) VolumeSpecificGetRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
```
Comment (kmova) [inline cmd/maya-apiserver/app/server/volume_endpoint.go]: should this and the below functions be -- volumeSpecificGetRequest ?

File: cmd/maya-apiserver/app/server/volume_endpoint.go [hunk 2]
```diff
@@ -87,23 +88,23 @@ func (s *HTTPServer) vsmList(resp http.ResponseWriter, req *http.Request) (inter
...
-func (s *HTTPServer) vsmRead(resp http.ResponseWriter, req *http.Request, vsmName string) (*v1.Volume, error) {
+// VolumeRead is the http handler that fetches the details of a Volume
+func (s *HTTPServer) VolumeRead(resp http.ResponseWriter, req *http.Request, volName string) (*v1.Volume, error) {
 
-	fmt.Println("[DEBUG] Processing Volume read request")
+	glog.Infof("Processing Volume read request")
 
-	if vsmName == "" {
+	if volName == "" {
 		return nil, CodedError(400, fmt.Sprintf("Volume name is missing"))
```
Comment (satyamz) [inline cmd/maya-apiserver/app/server/volume_endpoint.go]: I think you should log this error, Do you? There are other places like this!

File: cmd/maya-apiserver/app/server/volume_endpoint.go (hunk 1 above)
Comment (prateekpandey14) [reply cmd/maya-apiserver/app/server/volume_endpoint.go]: Yes it should be.

File: cmd/maya-apiserver/app/server/volume_endpoint.go (hunk 2 above)
Comment (prateekpandey14) [reply cmd/maya-apiserver/app/server/volume_endpoint.go]: `CodedError` accept only string right now as a arg, and glog don't have any error method which return string.

**Comment by kmova** [comment]
@AmitKumarDas  -- should we cover the logging as part of this issue : https://github.com/openebs/openebs/issues/818

**Pull request merged**